.drone.yml

```diff
@@ -494,9 +494,9 @@ steps:
     environment:
       # increment these variables when a new major/minor version is released to bump the automatic builds
       # this only needs to be done on the master branch, as that's the branch that the Drone cron is configured for
-      CURRENT_VERSION_ROOT: v5.0
-      PREVIOUS_VERSION_ONE_ROOT: v4.4
-      PREVIOUS_VERSION_TWO_ROOT: v4.3
+      CURRENT_VERSION_ROOT: v5.1
+      PREVIOUS_VERSION_ONE_ROOT: v5.0
+      PREVIOUS_VERSION_TWO_ROOT: v4.4
```

Comment:
I think it should be `v5.1`, `v4.4`, `v4.3` (since `5.1` is basically a patch release for `5.0` and not a new major release)

Reply:
I think that will probably make things worse, because we then won't be updating the 5.0 Docker image while we _are_ still updating the 4.4 and 4.3 images - it's likely that customers will get confused.

I can modify things to temporarily do 5.1, 5.0, 4.4 and 4.3 if that's what we want. Ideally we'd be doing 5, 4.4 and 4.3 but as we haven't communicated things externally yet, I think we need to stick with the current pattern until we do.

Reply:
Sounds good, as long as we keep 4.3 supported.
It should be supported until 6.0 comes out

Reply:
https://github.com/gravitational/teleport/pull/5221/commits/a376bf73b11f8006eeec1361b564f9aa276f458c adds support back in for building `v4.3`. Hopefully we don't release a `v5.2` before `v6.0` comes out, otherwise we'll start getting into `PREVIOUS_VERSION_FOUR` and beyond.